Makefile

```diff
@@ -0,0 +1,26 @@
+SOURCE=src/cisco_gnmi
+TEST_DIR=tests
+
+.PHONY: clean
+clean:
+	rm -rf .coverage htmlcov/
+
+.PHONY: create-env
+create-env:
+	python3 -m venv cisco_gnmi
```

Comment:
This is assuming exclusively Python 3 environment, as is we want to support Python 2.7 as well. Currently `pipenv` is being utilized for managing the project dependencies, `poetry` or `venv` could also be easily swapped in..
For the purposes of this pull we will want to refactor this.